frontend/www/js/omegaup/arena/contest_practice.ts

```diff
@@ -262,6 +263,14 @@ OmegaUp.on('ready', () => {
     },
   });
 
+  const locationOptions = getOptionsFromLocation(window.location.hash);
+
+  contestPractice.problem = locationOptions.problem;
+  contestPractice.popupDisplayed = locationOptions.popupDisplayed;
+  contestPractice.showNewClarificationPopup =
+    locationOptions.showNewClarificationPopup;
+  contestPractice.guid = locationOptions.guid;
```

Comment:
[esto](https://stackoverflow.com/a/43627653/688337) funciona?

```suggestion
  Object.assign(contestPractice, getOptionsFromLocation(window.location.hash));
```